src/tests/Microsoft.PowerFx.Core.Tests.Shared/ExpressionTestCases/AsType_UO.txt

```diff
@@ -0,0 +1,151 @@
+#SETUP: AllowTypeLiteral
+
+// Primitives
+>> AsType(ParseJSON("987654321"), Number)
+987654321
+
+>> AsType(ParseJSON("98765.4321"), Decimal)
+98765.4321
+
+>> AsType(ParseJSON("-1.3"), Decimal)
+-1.3
+
+>> AsType(ParseJSON("2e3"), Decimal)
+2000
+
+>> AsType(ParseJSON("98765.4321"), Number) > AsType(ParseJSON("98765.4320"), Number)
+true
+
+>> AsType(ParseJSON("""AstypeFunction"""), Type(Text))
+"AstypeFunction"
+
+>> AsType(ParseJSON("""1984-01-01"""), Date)
+Date(1984,1,1)
+
+>> AsType(ParseJSON("""1900-12-31T23:59:59.999Z"""), DateTime)
+DateTime(1900,12,31,15,59,59,999)
+
+>> AsType(ParseJSON("""1900-12-31T23:59:59.999"""), DateTime)
+DateTime(1900,12,31,23,59,59,999)
+
+>> AsType(ParseJSON("""1900-12-31T23:59:59.999+00:00"""), DateTime)
+DateTime(1900,12,31,15,59,59,999)
+
+>> AsType(ParseJSON("""1900-12-31T23:59:59.999-08:00"""), DateTime)
+DateTime(1900,12,31,23,59,59,999)
+
+>> AsType(ParseJSON("""1900-12-31"""), DateTime)
+DateTime(1900,12,31,0,0,0,0)
+
+>> AsType(ParseJSON("""1900-12-31T23:59:59.999"""), Date)
+Date(1900,12,31)
+
+>> AsType(ParseJSON("""1900-12-31T00:00:00.000Z"""), Date)
+Date(1900,12,31)
+
+>> AsType(ParseJSON("""11:59:59.999"""), Time)
+Time(11,59,59,999)
+
+>> AsType(ParseJSON("""00:00:00"""), Time)
+Time(0,0,0,0)
+
+>> AsType(ParseJSON("""1900-12-31T00:00:00.000Z"""), DateTimeTZInd)
```

Comment:
>DateTimeTZInd [](http://example.com/codeflow?start=53&length=13)

What does this type mean? Is it the same as UTC? If so, then the next test should not add 8 hours (Pacific Time adjustment). Sorry, but date/times are hard :-D